mne/gui/_viewer.py

```diff
@@ -222,28 +216,30 @@ def _on_hide(self):
     @on_trait_change('scene.activated')
     def _plot_points(self):
         """Add the points to the mayavi pipeline"""
-#         _scale = self.scene.camera.parallel_scale
+        from . import _testing_mode
 
         if hasattr(self.glyph, 'remove'):
             self.glyph.remove()
         if hasattr(self.src, 'remove'):
             self.src.remove()
 
+        _toggle_mlab_render(self, False)
         x, y, z = self.points.T
         scatter = pipeline.scalar_scatter(x, y, z)
-        glyph = pipeline.glyph(scatter, color=self.rgbcolor,
-                               figure=self.scene.mayavi_scene,
+        fig = self.scene.mayavi_scene if not _testing_mode() else None
```

Comment:
Our lines / calls are still covered. I think if it doesn't work there is some upstream `mayavi` bug.